### PR TITLE
Added youtube

### PR DESCRIPTION
**Pull request by FayasNoushad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ Initially created on a YouTube live stream <https://www.youtube.com/watch?v=Jorl
...
-1. Fork the project.
-2. Clone the project.
-3. Navigate to the project directory `cd LinkFree`.
-4. Install dependencies with `npm install`.
-5. Run `npm start`.
+1. Fork the project
+2. Clone the project
+3. Navigate to the project directory `cd LinkFree`
+4. Install dependencies with `npm install`
+5. Run `npm start`
```
Comment (eddiejaoude) [inline README.md:16]: These got added in a previous PR, I don't mind which style we have, but we can't have PRs adding/removing these all the time https://github.com/EddieHubCommunity/LinkFree/commit/0caf1081dc270f555fe73aa9a30cea8df8d91757#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

**Review by eddiejaoude — APPROVED** [review]
I would suggest having these as separate PRs so they do not block each other. But I will accept both this time

**Comment by FayasNoushad** [comment]
> I would suggest having these as separate PRs so they do not block each other. But I will accept both this time

Eddie, Please add "hacktoberfest-accepted" label.

**Comment by nhcarrigan** [comment]
PRs to add/update your `.json` file aren't eligible for the label as they aren't in line with the core values of Hacktoberfest.

**Comment by FayasNoushad** [comment]
> PRs to add/update your `.json` file aren't eligible for the label as they aren't in line with the core values of Hacktoberfest.

Ok

**Pull request merged**